packages/sit-onyx/src/components/TestInput/TestInput.vue

```diff
@@ -34,7 +34,7 @@ const handleChange = (event: Event) => {
 </script>
 
 <template>
-  <label>
+  <label class="input__wrapper">
```

Comment:
as I did in my PR (which is still draft), BEM requires this class to be named

```suggestion
  <label class="input">
```